forgit.plugin.zsh

```diff
@@ -21,8 +21,9 @@ FORGIT="$INSTALL_DIR/bin/git-forgit"
 # backwards compatibility:
 # export all user-defined FORGIT variables to make them available in git-forgit
 unexported_vars=0
+[[ -n "$BASH_VERSION" ]] && set -o posix
```

Comment:
Maybe it would be good to add a describing comment here why we're calling `set -o posix`.